client/src/App.js

```diff
@@ -4,17 +4,25 @@ import ContactUs from "./pages/ContactUs";
 import Header from "./Components/Header";
 import Footer from "./Components/Footer";
 import Dashboard from "./pages/Dashboard";
+import Subscription from "./pages/Subscription";
+import SignUpForm from "./pages/SignUpForm";
+import useAuth from "./customHooks/useAuth";
 
-const App = () => (
-	<div>
-		<Header />
-		<Routes>
-			<Route path="/" element={<Home />} />
-			<Route path="/dashboard" element={<Dashboard />} />
-			<Route path="/contact-us" element={<ContactUs />} />
-		</Routes>
-		<Footer />
-	</div>
-);
+const App = () => {
+	const { isLoggedIn } = useAuth();
+	return (
+		<div>
+			<Header />
+			<Routes>
+				<Route path="/" element={<Home />} />
+				{isLoggedIn && <Route path="/sign-up" element={<SignUpForm />} />}
```

Comment:
SHouldn't this be when they are not logged in?

`{!isLoggedIn && <Route path="/sign-up" element={<SignUpForm />} />}`

Reply:
It's not signUp to use the app, It's sign up for become a provider.
It's for people try to access this page from url.

Reply:
At the moment, we have some bugs. Nearly end of next week, we should go over them.